libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -416,8 +416,13 @@ const AP_Param::GroupInfo NavEKF3::var_info[] = {
     // @Units: m
     AP_GROUPINFO("NOAID_M_NSE", 35, NavEKF3, _noaidHorizNoise, 10.0f),
 
-    // 36 was LOG_MASK, used for specifying which IMUs/cores to log
-    // replay data for
+    // @Param: BETA_MASK
+    // @DisplayName: Bitmask controlling sidelip angle fusion
+    // @Description: 1 byte bitmap controlling use of sideslip angle fuson for estimation of non wind states during operation of 'fly forward' vehicle types. Use of sideslip fusion to update non wind states is performed by default when dead reckoning and cannot be didsabled. 
```

Comment:
typo on "disabled"
It would be good to add a sentence to explain why a user might enable this option

Reply:
I've pushed a couple of commits to address these issues.